src/logic/operate.js

```diff
@@ -20,5 +20,5 @@ export default function operate(numberOne, numberTwo, operation) {
       return one.div(two).toString();
     }
   }
-  throw Error(`Unknown operation '${operation}'`);
+  throw Error(`Unknown operation performed '${operation}'`);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made to the error message is a good one as it provides more context about what operation was attempted when the error occurred. However, the wording could be improved for clarity.
```diff
- throw Error(`Unknown operation performed '${operation}'`);
+ throw Error(`Operation '${operation}' not recognized`);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->